src/components/Accordion/FormAccordion.tsx

```diff
@@ -58,21 +59,36 @@ const HeaderWrapper = styled.div`
   }
 `;
 
+const StyledAccordionContent = styled(AccordionContent)`
+  &[data-wide='true'] {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+
+    > div {
+      width: 1200px !important;
+      left: 0 !important;
+    }
+  }
+`;
+
```

Comment:
Tenker at alt som gjelder styling og utviding av størrelse for frontpage artikler burde flyttes til egen PR. Kanskje lettere å fokusere kun på Grid i denne?

Reply:
Er det så mye styling som kreves?

Reply:
Tenker det er lettere mtp. å splitte hva PR'en handler om. Men kan godt ta det her